enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/grpc/ExtAuthService.java

```diff
@@ -161,7 +161,9 @@ private CheckResponse buildResponse(CheckRequest request, ResponseObject respons
             }
 
             if (responseObject.getHeaderMap() != null) {
-                responseObject.getHeaderMap().forEach((key, value) -> {
+                Map<String, String> headersMap = responseObject.getHeaderMap();
+                headersMap.put("x-wso2-ratelimit-api-policy", "default");
```

Comment:
If we are not going to support advanced rate limitings, maybe we can remove this. We may not need Enforcer for rate limiting.